UNRELEASED.md

```diff
@@ -9,6 +9,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Bug fixes
 
 - Fixed `Autocomplete` empty state example Markdown not parsing correctly ([#592](https://github.com/Shopify/polaris-react/pull/592))
+- Fixed `TopBar`'s `UserMenu` from being left-aligned when `TopBar` isn't passed a `searchField` prop. ([#597](https://github.com/Shopify/polaris-react/pull/597))
```

Comment:
```suggestion
- `TopBar`'s `UserMenu` is now right-aligned when `TopBar` isn't passed a `searchField` prop. ([#597](https://github.com/Shopify/polaris-react/pull/597))
```

I find "it is now right aligned" a bit easier to parse and more meaningful "it is no-longer left aligned".